### DISCUSSION TITLE
Is GH discussions a good platform for ksconf topics?

### DISCUSSION THREAD

**Discussion by lowell80** [Kintyre/ksconf#82] [General]
@SID800, @vaniovanio, @Jalkar, @nebffa, @mthambipillai, @jonesy1234, @tiberiustribun

As issue creators and contributors, I wanted to get some feedback on the [Python 2.7](https://github.com/Kintyre/ksconf/discussions/81) topic, but more broadly:  I haven't used the discussions feature on GitHub before and don't know how well it works or if it's a good platform for these types of conversations.  And even more broadly, I really don't have a good feel for how many people are actively using ksconf and for what, so I thought this was maybe the discussions feature could be helpful in general.  We have a channel on the Splunk Users' channel, but the history get's archived so quickly that it doesn't end up being helpful to reference.

Is having a place to discuss ksconf topics (rather than just bug reports, and/or technical questions) something that would be helpful?

**Comment by nebffa** [comment]
Is this a new feature of GitHub? I haven't noticed it before. Nice - I'll add a comment on your Python 2.7 issue. As for general discussion, it probably wouldn't be helpful for me. I just use ksconf for my job so I only look at things when I need to.

**Comment by lmnogues** [comment]
GH will be better than Slack for those question... Splunk should pay for the Community Slack to avoid the auto archive of message ^^
